### PR TITLE
Add Browser platform support 

### PR DESCRIPTION
**Pull request by reytum**
1) Added support for browser platform in cordova
2) Removed swift support plugin which was incompatible with cordova 9. Have merged changes from the plugin to kommunicate plugin.

### CODE REVIEW

File: src/browser/KommunicateCordovaPluginProxy.js [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+	},
+
+	logout: function (successCallback, errorCallback) {
+		console.log("Called logout function ");
+		if (isUserLoggedIn() && typeof Kommunicate != 'undefined' && Kommunicate) {
+			init((response) => {
+				//Kommunicate.launchConversation();
+				Kommunicate.logout();
+				localStorage.removeItem('KM_PLUGIN_USER_DETAILS');
+				location.reload()
```
Comment (devashishmamgain) [inline src/browser/KommunicateCordovaPluginProxy.js]: @reytum is this required?

File: src/browser/KommunicateCordovaPluginProxy.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+					if (kmUser.withPreChat == true) {
+						kmUser.userId = JSON.parse(sessionStorage.getItem("mckAppHeaders")).userId;
+          }
+          
+          KommunicateGlobal.document.getElementById('km-chat-widget-close-button').addEventListener('click',function(){
+            var testClick = parent.document.getElementById("kommunicate-widget-iframe");
+            testClick.style.display = "none";
+          });
+
+					console.log("Login response : " + JSON.stringify(response));
```
Comment (devashishmamgain) [inline src/browser/KommunicateCordovaPluginProxy.js]: @reytum can this be disabled, some customers have raised concern about printing user's info in console

Comment (reytum) [reply src/browser/KommunicateCordovaPluginProxy.js]: Will remove the logs as there is no option to restrict logs for productions and localhost in web.

File: src/browser/KommunicateCordovaPluginProxy.js (hunk 1 above)
Comment (reytum) [reply src/browser/KommunicateCordovaPluginProxy.js]: location.reload can be removed.

Comment (reytum) [reply src/browser/KommunicateCordovaPluginProxy.js]: Its used to clear the Kommunicate variable as if someone logs in just after logging out the error comes : script already loaded.

**Pull request merged**